vtStorAta/Makefile

```diff
@@ -0,0 +1,72 @@
+# Compiler, tools and options
+CC            = gcc
+CXX           = g++
+CFLAGS        = -pipe -O2 -Wall -W -D_REENTRANT -fPIC 
+CXXFLAGS      = -pipe -std=c++14 -fpermissive -O2 -Wall -W -D_REENTRANT -fPIC 
+
+# Define the directories.
+DEST_DIR                        = ../Debug
+OBJECTS_DIR                     = $(DEST_DIR)/vtStorAta
+EXTERN_DIR                      = ../Extern
+COMMON_DIR                      = ../Common
+COMMON_PF_X86X64_DIR            = ../Common/Platform/x86x64
+COMMON_PF_X86X64_LINUX_DIR      = ../Common/Platform/x86x64/Linux
+VTSTOR_DIR                      = ../vtStor
+VTSTOR_PF_LINUX_DIR             = ../vtStor/Platform/Linux
+STORAGEUTILITY_DIR              = ../StorageUtility
+STORAGEUTILITY_LINUX_DIR        = ../StorageUtility/Linux 
+VTSTORATA_DIR                   = ../vtStorAta
+VTSTORATA_PF_LINUX_DIR          = ../vtStorAta/Platform/Linux
+VTSTORATAPROTOCOL_DIR           = ../vtStorAtaProtocol
+VTSTORATAPROTOCOL_PF_LINUX_DIR  = ../vtStorAtaProtocol/Platform/Linux  
+
+INCPATH       = -I. -I$(EXTERN_DIR) -I$(COMMON_DIR) -I$(COMMON_PF_X86X64_DIR) -I$(COMMON_PF_X86X64_LINUX_DIR) -I$(STORAGEUTILITY_DIR) -I$(STORAGEUTILITY_LINUX_DIR) -I$(VTSTOR_DIR) -I$(VTSTOR_PF_LINUX_DIR) -I$(DEST_DIR) \
+                -I$(VTSTORATA_DIR) -I$(VTSTORATA_PF_LINUX_DIR) -I$(VTSTORATAPROTOCOL_DIR) -I$(VTSTORATAPROTOCOL_PF_LINUX_DIR) 
+LINK          = g++
+LFLAGS        = -Wl,-O1 -shared -Wl,-soname,libvtStorAta.so.1
+LIBS          = $(SUBLIBS)  -L/usr/lib/arm-linux-gnueabihf -ludev -L$(DEST_DIR)/ -lStorageUtility -lvtStor -lvtStorAtaProtocol 
```

Comment:
please check "arm-linux-gnueabihf" if remove it then have no errors. Please remove. On other platforms (like Unix) will can get error because have no the lib